src/main/resources/templates/admin.html

```diff
@@ -12,61 +12,65 @@
             <div class="sightings-pending-container">
 
                 <h1>Pending Sightings</h1>
+
                 <p>Only members with the Admin role should be able to get here!</p>
+
                 <div th:if="${param.error}">
                     There was an error loading the pending sightings.
                 </div>
-                <!-- Place a div if no sight is pending to indicate no work to do. Use thymeleaf to check no sight pending -->
 
-                <form th:action="@{/admin}" method="post" th:object="${sightings}">
-                    <!-- loop throught sighting pending to be approved-->
-                    <div class="sightings-pending-admin-form">
+                <div th:if="${sightings==null and sightings.empty}">
```

Comment:
Why do we need to check for sightings.empty as well as sightings==null?

Reply:
I ran the code with no sightings in my database, and I did not see this message, so I think there is a bug here?

Reply:
I will have to invest this with a new ticket. I will remove null, I added it in case the object it is not created.